static/files/forms-data.json

```
@@ -0,0 +1,266 @@
+{
+    "forms": [
+        {
+            "formData": {
+                "title": "Test form",
+                "introText": "Intro text",
+                "formId": "1266",
+                "returnUrl": "/appliance/thank-you",
+                "product": ""
+            },
```

```suggestion
    "forms": {
       "/path/to/page": {
            "formData": {
                "title": "Test form",
                "introText": "Intro text",
                "formId": "1266",
                "returnUrl": "/appliance/thank-you",
                "product": ""
            },
```
You could use a sort of hash map here to access the form data. This could be the page path as there is usually on one form per page